### ISSUE TITLE
How to design multi language pouch?

### ISSUE THREAD

**Issue by hashg** [nolanlawson/ember-pouch#178]
I'm trying to build mobile app with multi language support. 

- Has anyone in this community tried it?
- What are the techniques used. 
- Lessons learnt?

**Comment by broerse** [comment]
@hashg I have not tried this. But take a look at https://github.com/jamesarosen/ember-i18n and perhaps ask on https://discuss.emberjs.com

**Comment by hashg** [comment]
@broerse You are correct, I'm using ember-i18n for static content. But how to store and retrieve language specific data in DB?

**Comment by srsgores** [comment]
I prefer ember-intl, which is a more comprehensive solution to localisation.

**Comment by broerse** [comment]
@hashg is this not a normal hasMany relation? `translations: DS.hasMany('translations:')`
